packages/eas-cli/src/run/android/aapt.ts

```diff
@@ -0,0 +1,52 @@
+import spawnAsync, { SpawnResult } from '@expo/spawn-async';
+import glob from 'fast-glob';
+
+import Log from '../../log';
+import { sdkRoot } from './sdk';
+
+async function aaptAsync(...options: string[]): Promise<SpawnResult> {
+  try {
+    return await spawnAsync(await getAaptExecutableAsync(), options);
+  } catch (error: any) {
+    const errorMessage = (error.stderr || error.stdout || error.message).trim();
+    error.message = errorMessage;
```

Comment:
```suggestion
    if (error.stderr) {
      Log.error(error.stderr);
      throw error;
    }
```
Maybe that would be better?
Note that this approach will not be correct if you are catching aapt errors in some cases